File: app/src/main/java/com/mdmobile/cyclops/dataModel/api/AntiVirus.java
package com.mdmobile.cyclops.dataModel.api;

/**
 * Represent AntiVirus Gson class
 */


public class AntiVirus {

    private String AntivirusDefinitionsVersion, LastEmptyQuarantine, LastVirusDefUpdate, LastVirusScan;

    public AntiVirus(String antivirusDefinitionsVersion, String lastEmptyQuarantine, String lastVirusDefUpdate, String lastVirusScan) {
        AntivirusDefinitionsVersion = antivirusDefinitionsVersion;
        LastEmptyQuarantine = lastEmptyQuarantine;
        LastVirusDefUpdate = lastVirusDefUpdate;
        LastVirusScan = lastVirusScan;
    }

    public String getAntivirusDefinitionsVersion() {
        return AntivirusDefinitionsVersion;
    }

    public String getLastEmptyQuarantine() {
        return LastEmptyQuarantine;
    }

    public String getLastVirusDefUpdate() {
        return LastVirusDefUpdate;
    }

    public String getLastVirusScan() {
        return LastVirusScan;
    }
}

File: app/src/main/java/com/mdmobile/cyclops/dataModel/api/devices/test/Antivirus.kt
package com.mdmobile.cyclops.dataModel.api.devices.test

import com.google.gson.annotations.SerializedName

data class Antivirus(
        @SerializedName("AntivirusDefinitionsVersion") val antivirusDefinitionsVersion: String = "",
        @SerializedName("LastEmptyQuarantine") val lastEmptyQuarantine: Int = 0,
        @SerializedName("LastVirusDefUpdate") val lastVirusDefUpdate: Int = 0,
        @SerializedName("LastVirusScan") val lastVirusScan: Int = 0
)